app/assets/javascripts/controllers/security_group/security_group_form_controller.js

```diff
@@ -20,33 +18,27 @@ ManageIQ.angular.app.controller('securityGroupFormController', ['securityGroupFo
     vm.newRecord = securityGroupFormId === "new";
     vm.saveable = miqService.saveable;
 
-    if (vm.newRecord) {
-      vm.afterGet = true;
-      vm.modelCopy = angular.copy( vm.securityGroupModel );
-    } else {
-      miqService.sparkleOn();
-      API.get("/api/security_groups/" + securityGroupFormId + "?attributes=name,ext_management_system.name,description,cloud_tenant.name,firewall_rules").then(function(data) {
-        vm.securityGroupModel.name = data.name;
-        vm.securityGroupModel.ems_name = data.ext_management_system.name;
```

Comment:
It looks like you're removing `ems_name` completely - is that intentional?

The same goes for `cloud_tenant_name`.

Reply:
The name is only for the display.